### PR TITLE
1559: Adds dependency on 2718

### PR DESCRIPTION
**Pull request by MicahZoltu**
Integrates 2718 (Typed Transactions) into EIP-1559.  Overall a pretty simple/small change, mostly with just some adjustments to pre-processing.

I took the opportunity to also rename some of the variables both to improve clarity and to reduce conflict in the community (bribe vs inclusion fee).  If someone feels strongly about it, I can split out the 2718 changes and the variable name changes.

### CODE REVIEW

**Comment by MicahZoltu** [comment]
Note: This shouldn't be merged until after #3204, and it will need a fixup after that is merged.

**Review by yorickdowne — CHANGES_REQUESTED** [review]
"and they will pay the inclnusion fee per gas set in the transaction," minor typo, should be "and they will pay the inclusion fee per gas set in the transaction,"

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - EIP 1559 requires approval from one of (@mslipper, @afdudley, @econoar, @i-norden, @abdelhamidbakhta, @vbuterin)

File: EIPS/eip-1559.md
Comment (MicahZoltu) [inline EIPS/eip-1559.md:22]:
```suggestion
The transaction will always pay the base fee per gas of the block it was included in, and they will pay the inclusion fee per gas set in the transaction, as long as the combined amount of the two fees doesn't exceed the transaction's maximum fee per gas.
```

**Review by abdelhamidbakhta — APPROVED** [review]
Looks good to me.

**Pull request merged**
